docs/publications/BioHackrXiv/paper.md

```diff
@@ -0,0 +1,215 @@
+These are described in more detail below.
+
+
+## Prototype interactive table for Galaxy communities
+
+The described workflow for the Galaxy tool metadata extractor (see Figure \ref{metadata_extractor_pipeline}) was successfully implemented ([GitHub repository](https://github.com/galaxyproject/galaxy_tool_extractor)) and could more then 1,300 Galaxy tool suites (see the [GitHub repository pages](https://galaxyproject.github.io/galaxy_tool_metadata_extractor) for an up-to-date table).
+Of those tool suites, only 267 had a bio.tools identifier, which highlights the importance of performing the annotation process in parallel and complementing the tools with additional metadata. 
+An example view of the created interactive table is shown in Figure \ref{web_table}. 
+As mentioned above, the filtered table for the microGalaxy community has already been embedded in the [Hub page for microGalaxy](https://galaxyproject.org/community/sig/microbial#tools), as well as the dedicated [microGalaxy subdomain](https://microgalaxy.usegalaxy.eu/). 
+The process for creating a new interactive table for a community is currently being transformed into a GTN tutorial.
+
+![Screenshot of the interactive web table. The table provides comprehensive metadata for all Galaxy wrappers of a specific community and allows for custom searches based on logic filters over all columns. In the shown example, the user queries for all up-to-date tools that are annotated with an EDAM operation that includes "assembly". \label{web_table}](./figures/itable.png)
+
+
+## bio.tools and EDAM annotations for microbiome community
+
+During the week of the BioHackathon, the microGalaxy community executed the annotation workflow as described in the Methods section (see also Figure \ref{annotation_workflow}). 
+The initial filtered tool table of the microGalaxy included 218 tool suites, of which 61 had corresponding bio.tools identifiers. 
+The progress of the work was tracked using a GitHub project board. 
+After the annotation process, the number of tools with bio.tools annotations was increased to 107. 
+The added annotations for each respective bio.tools entry were also collectively reviewed by the team. 
+A rerun of the Galaxy tool metadata extractor pipeline collected the additional information, and the metadata is now included in the interactive microGalaxy tool table. 
+
+
+## Training materials and updates to standards
+
+To provide the Galaxy research communities with simple and straightforward guide to annotating their respective tool stacks, the described annotation workflow is currently being transformed into a GTN tutorial. 
```

Comment:
The tutorial is ready to be merged: https://github.com/galaxyproject/training-material/pull/4734

Reply:
There is also a tutorial for community Galaxy tool table creation ready: https://github.com/galaxyproject/training-material/pull/4740